content/blog/logging-docker-elk.md

```diff
@@ -0,0 +1,93 @@
+---
+author: Mark Woodbridge
+date: 2021-02-15
+title: Aggregating logs from services deployed with Docker
+tags:
+- Docker
+- ELK
+- Logging
+---
+
+# Introduction
+
+The [Elastic (ELK) Stack](https://www.elastic.co/what-is/elk-stack) is commonly used to centralise logs from across
```

Comment:
Something that might be useful here is some motivating reason as to _why_ one might want to go through the hassle of setting up a logging server; what are we trying to make easier? This sort of starts from the point of view of already wanting one

Reply:
Done